src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+supports communication among devices that implement Bluetooth 4.0 or later.
+
+When a website requests access to nearby devices using
+`navigator.bluetooth.requestDevice`, the browser will prompt user with a device
+chooser where they can pick one device or simply cancel the request.
+
+
+<figure class="w-figure">
+  <video controls autoplay loop muted>
+    <source src="https://storage.googleapis.com/web-dev-assets/bluetooth/bluetooth-device-chooser.mp4">
+  </video>
+  <figcaption class="w-figcaption">Bluetooth device user prompt.</figcaption>
+</figure>
+
+
+The `navigator.bluetooth.requestDevice` function takes a mandatory Object that
+defines filters. These filters are used to return only devices that match some
+advertised Bluetooth GATT services and/or the device name.
+
+For instance, requesting Bluetooth devices advertising the [Bluetooth GATT
+Battery Service] is this simple:
+
+```js
+navigator.bluetooth.requestDevice({ filters: [{ services: ['battery_service'] }] })
+.then(device => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+If your Bluetooth GATT Service is not on the list of [the standardized Bluetooth
+GATT services] though, you may provide either the full Bluetooth UUID or a short
+16- or 32-bit form.
+
+```js
+navigator.bluetooth.requestDevice({
+  filters: [{
+    services: [0x1234, 0x12345678, '99999999-0000-1000-8000-00805f9b34fb']
+  }]
+})
+.then(device => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+You can also request Bluetooth devices based on the device name being advertised
+with the `name` filters key, or even a prefix of this name with the `namePrefix`
+filters key. Note that in this case, you will also need to define the
+`optionalServices` key to be able to access some services. If you don't, you'll
+get an error later when trying to access them.
+
+```js
+navigator.bluetooth.requestDevice({
+  filters: [{
+    name: 'Francois robot'
+  }],
+  optionalServices: ['battery_service']
+})
+.then(device => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+Finally, instead of `filters` you can use the `acceptAllDevices` key to show all
+nearby Bluetooth devices. You will also need to define the `optionalServices`
+key to be able to access some services. If you don't, you'll get an error later
+when trying to access them.
+
+```js
+navigator.bluetooth.requestDevice({
+  acceptAllDevices: true,
+  optionalServices: ['battery_service']
+})
+.then(device => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+{% Aside 'caution' %}
+This may result in a bunch of unrelated devices being shown in the chooser and
+energy being wasted as there are no filters. Use it with caution.
+{% endAside %}
+
+### Connect to a Bluetooth Device
+
+So what do you do now that you have a `BluetoothDevice` returned from
+`navigator.bluetooth.requestDevice`'s Promise? Let's connect to the Bluetooth
```

Comment:
```suggestion
So what do you do now that you have a `BluetoothDevice`? Let's connect to the Bluetooth
```
Nitpick: Can we avoid the inline code followed by "'s"? (`navigator.bluetooth.requestDevice`'s)

It looks awkward:

![image](https://user-images.githubusercontent.com/4713486/96657625-860aaa00-12f7-11eb-8e2a-9e75e7f98ac8.png)

Reply:
Done.